relatio/_embeddings.py

```diff
@@ -130,22 +143,31 @@ def get_vector(self, phrase: str) -> Optional[np.ndarray]:
     def _get_default_vector(self, phrase: str) -> np.ndarray:
         return self._embeddings_model._get_default_vector(phrase)
 
+    # This will require refactoring for speed (in the case of spacy and USE)
+    def _get_vectors(self, phrases: str, progress_bar: bool = False) -> np.ndarray:
+
+        if progress_bar:
+            print("Computing phrase embeddings...")
+            phrases = tqdm(phrases)
+
+        vectors = []
+        for i, phrase in enumerate(phrases):
+            vector = self.get_vector(phrase)
+            vectors.append(np.array([vector]))
+        vectors = np.concatenate(vectors)
+        return vectors
```

Comment:
This should not give a big improvement. We should check it on an example.

`_get_vectors` can shine one it is specialized. So I suggest to add it to the Base if possible as it was some time ago.

Reply:
Agreed for specialized _get_vectors(). 
PS: It's a "big" improvement because we use np.concatenate() only once.

Reply:
I wrote it mainly to keep the NarrativeModel() code clear. 
In narrative_models.py, now a programmer only sees:
`vectors = self.embeddings_model._get_vectors(phrases)`
I think it makes the final code clearer.

Reply:
I will take care of it.